src/features/Classes/ClassesFilters/index.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Col, Form } from '@edx/paragon';
+import { Select, Button } from 'react-paragon-topaz';
+
+import { fetchAllCourses, fetchAllClassesData } from 'features/Common/data';
+import { updateCurrentPage, updateFilters } from 'features/Classes/data/slice';
+import { getClasses } from 'features/Classes/data';
+
+import { RequestStatus, initialPage } from 'features/constants';
+
+const initialStateTable = {
```

Comment:
Well... Technically we aren't using this for the state of the table directly, so what about:

```suggestion
const initialRequestParams = {
```